npm-debug.log

```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```

Comment:
don't include your debug log. Even maybe add `npm-debug.log` to the `.gitignore` so this doesn't happen again.